CrosswordMe.py

```diff
@@ -1,46 +1,98 @@
 from random import randint
+from random import sample
 
-#input
+def print_puzzle(puzzle):
+    #Prints the puzzle puzzle on a board
+    for line in range(line_count):
+        for position in range(column_count):
+            print(str(puzzle[line][position]) + " ", end="") #Prints character in current position
+        print() #Prints Newline
 
-words = ["one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
-user_input = ""
-print(len(words))
+def word_input():
+    #Collects the words for the puzzle
+    words = ["YEAHAA"] * 5
+    #, "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten"]
+    user_input = ""
+    """while 1 == 1:
+        user_input = input("Word: ")
+        if user_input.lower() == "done":
+            break
+        words.append(user_input)"""
+    return words
 
-"""while 1 == 1:
-    user_input = input("Word: ")
-    if user_input.lower() == "done":
-        break
-    words.append(user_input)"""
+def define_field_size():
+    #defines field size
+    global line_count
+    line_count = 12
+    global column_count
+    column_count = 12
 
-# creating the puzzle
-## horizontals
-line = randint(0,9) #First line to write in
-lines_used = [] #Empty array which contains lines which have been written in already (can probably be removed when its possible to write multiple words in one line)
+def calculate_postions(word_length):
```

Comment:
Diese Funktion zum Beispiel ist ziemlich sauber gelößt.

Die Kommentare in den folgenden Zeilen sind aber ein bisschen überflüssig. Die Variablennamen sind aussagekräftiger.